runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimerInternals.java

```diff
@@ -114,14 +122,20 @@ public void setTimer(
       Instant timestamp,
       Instant outputTimestamp,
       TimeDomain timeDomain) {
-    timers.put(
-        getTimerDataKey(timerId, timerFamilyId),
-        namespace,
-        TimerData.of(timerId, timerFamilyId, namespace, timestamp, outputTimestamp, timeDomain));
+    TimerData timer =
```

Comment:
Is it possible to combine the overrides here? Seems like a minor risk to have a bunch of methods that have to have corresponding changes here. I am reacting to the addition of the `onModified` callback to each of them. Seems like one centralized method would be good if possible.

Reply:
Yeah - done